<launch>

    <!-- list of arguments that can be given as inputs of the launch file -->
    <arg name="step_file_path"   default="/input_step_files/test.step" />
    <arg name="output_folder_path"   default="/output_ros_urdf_packages" />
    <arg name="urdf_package_name"   default="test3" />

    <!-- Node wraped code -->
    <node pkg="urdf_from_step" name="urdf_creator" type="create_urdf.py" output="screen" required="true">

         <param name="step_file_path"   type="string" value="$(arg step_file_path)" />
         <param name="output_folder_path"   type="string" value="$(arg output_folder_path)" />
         <param name="urdf_package_name"   type="string" value="$(arg urdf_package_name)" />
         
    </node>

</launch>